autoload-config.sample.py

```diff
@@ -0,0 +1,15 @@
+FLOW_PORTAL_SITE = 'https://learn.concord.org'
```

Comment:
Do we need to worry about the staging version of dataflow using the staging version of learn?
When users login to https://dataflow.staging.concord.org/ext/flow it will redirect them to https://learn.staging.concord.org/